app/src/main/java/com/github/polypoly/app/ui/game/HeadsUpDisplay.kt

```diff
@@ -365,6 +369,7 @@ fun HudButton(testTag: String, onClick: () -> Unit, icon: ImageVector, descripti
             .testTag(testTag)
             .padding(bottom = Padding.large),
         shape = CircleShape,
+        elevation = ButtonDefaults.elevation(0.dp),
```

Comment:
does an elevation of 0 does anything?

Reply:
Yes, it removes the drop down shadow from the buttons to give them, like the other UI elements, a flat design feel as @tamaraaPrs suggested we do 😊